protocol/query/method/message/data/dataCashTransaction.json

```
@@ -0,0 +1,108 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://raw.githubusercontent.com/dedis/popstellar/master/protocol/query/method/message/data/dataPostTransaction.json",
+  "title": "Post a digital cash transaction",
+  "type": "object",
+  "properties": {
+    "object": {
+      "const": "transaction"
+    },
+    "action": {
+      "const": "post"
+    },
+    "transaction": {
+      "type": "object",
+      "description": "the transaction object",
+      "properties": {
+        "Version": {
+          "type": "integer",
+          "description": "The version of the transaction inputs"
+        },
+        "TxIn": {
+          "type": "array",
+          "description": "[Array[Objects]] array of output transactions to use as inputs",
+          "items": {
+            "type": "object",
+            "description": "Object representing a transaction output to use as an input for this transaction",
+            "properties": {
+              "TxOutHash": {
+                "type": "string",
+                "contentEncoding": "base64",
+                "description": "Previous (to-be-used) transaction hash"
+              },
+              "TxOutIndex": {
+                "type": "integer",
+                "description": "index of the previous to-be-used transaction"
+              },
+              "Script": {
+                "type": "object",
+                "description": "The script describing the unlock mechanism",
+                "properties": {
+                  "Type": {
+                    "type": "string",
+                    "description": "Type of script"
+                  },
+                  "Pubkey": {
+                    "type": "string",
+                    "contentEncoding": "base64",
+                    "description": "The recipient’s public key"
+                  },
+                  "Sig": {
+                    "type": "string",
+                    "contentEncoding": "base64",
+                    "description": "Signature on all txins and txouts using the recipient's private key"
+                  }
+                }
+              },
+              "Sequence": {
+                "type": "bytes",
+                "length": "4",
```

```suggestion
                "type": "integer",
```

I think `bytes` isn't a type:
`$.transaction.TxIn[0].Sequence: object found, unknown expected`